process/factory/metachain/vmContainerFactory.go

```diff
@@ -127,7 +127,24 @@ func (vmf *vmContainerFactory) Create() (process.VirtualMachinesContainer, error
 	return container, nil
 }
 
-func (vmf *vmContainerFactory) createSystemVM() (vmcommon.VMExecutionHandler, error) {
+// CreateForGenesis sets up all the needed virtual machines returning a container of all the VMs to be used in the genesis process
```

Comment:
add comment that genesis contains only a,b,c and d contracts

Reply:
done